src/core/dfn-finder.js

```diff
@@ -150,10 +150,14 @@ function findNormalDfn(defn, parent, name) {
  * @param {string} parent
  * @param {string} name
  */
-function decorateDfn(dfn, defn, parent, name) {
+export function decorateDfn(dfn, defn, parent, name) {
   if (!dfn.id) {
-    const middle = parent ? `${parent}-` : "";
-    const last = name.replace(/[()]/g, "").replace(/\s/g, "-");
+    const middle = parent ? `${parent.toLowerCase()}-` : "";
+    let last = name
+      .toLowerCase()
+      .replace(/[()]/g, "")
+      .replace(/\s/g, "-");
+    if (last === "") last = "the-empty-string";
```

Comment:
This part is always tricky 😄

Reply:
yeah, agree... I thought to just run with it. If it needs something more robust, can fix if it exhibits issues.

Reply:
Yeah, I'm okay with it for now.